src/store/list-data/request-action-mapping.ts

```diff
@@ -1,4 +1,4 @@
-import { RequestActionMappingI } from '@/models'
+import { RequestActionMappingI } from '@/interfaces'
```

Comment:
Oh this list data should totally not be in the store.

Perhaps a resources/ folder instead?

Reply:
For sure. I pulled all the `list-data` out of the module as well, although i did move it into it's own files here for now. 
Would probably make sense to remove the directory from store completely.